docs/examples/page-header/additional-sections.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <el-page-header :icon="null">
+  <el-page-header :icon="''">
```

Comment:
Why not `null`?

Reply:
Change to undefined.

![screenshots](https://github.com/element-plus/element-plus/assets/45450994/da2f9d68-a38d-4fd0-81ce-0a37b6ca438b)